pom.xml

```diff
@@ -18,15 +18,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
```

Comment:
This pom finds the kafka-proxy-*/poms though they have been moved down a dir?

Reply:
Yup. Defining the module as `path/to/the/real_module_name` works just fine in the parent pom, and then in the `real_module_name` pom.xml, we just unwind the `relativePath` to the parent. Can also undo this if you think it's unnecessary. I thought this would be nice once we have `kafka`, `rest`, `thrift1`, `thrift2`, etc as top-level directories.